app/scripts/controllers/createFromURL.js

```diff
@@ -0,0 +1,191 @@
+        type: "error",
+        message: namespace ?
+                  "Resources from the namespace \"" + namespace + "\" are not permitted." :
+                  "A valid namespace is required."
+      };
+    };
+
+    var alertInvalidTemplate = function(template, e) {
+      $scope.alerts.invalidTemplate = {
+        type: "error",
+        message: "The requested template \"" + template + "\" is not valid.",
+        details: "Reason: " + $filter('getErrorDetails')(e)
+      };
+    };
+
+    var alertResourceRequired = function() {
+      $scope.alerts.resourceRequired = {
+        type: "error",
+        message: "A valid image stream or template is required."
+      };
+    };
+
+    var showInvalidResource = function() {
+      $scope.alerts.invalidResource = {
+        type: "error",
+        message: "Image streams and templates cannot be combined."
+      };
+    };
+
+    var getTemplateParamsMap = function () {
+      try {
+        return JSON.parse($routeParams.templateParamsMap);
+      }
+      catch (e) {
+        $scope.alerts.invalidTemplateParams = {
+          type: "error",
+          message: "The template parameters are not valid."
+        };
+      }
+    };
+
+    var namespaceWhitelist = window.OPENSHIFT_CONSTANTS.CREATE_FROM_URL_WHITELIST;
+    var whiteListedCreateDetailsKeys = ['namespace', 'app', 'imageStream', 'imageTag', 'sourceURI', 'sourceRef', 'contextDir', 'template', 'templateParamsMap'];
+    var createDetails = _.pick($routeParams, function(value, key) {
+      // routeParams without a value (e.g., ?app&) return true, which results in "true" displaying in the UI
+      return _.contains(whiteListedCreateDetailsKeys, key) && (value && value !== true);
+    });
+    // if no namespace is specified, set it to 'openshift'
+    createDetails.namespace = createDetails.namespace || 'openshift';
+
+    var getResources = function() {
+      if (createDetails.imageStream) {
+        DataService
+          .get("imagestreams", createDetails.imageStream, {namespace: createDetails.namespace}, {
+            errorNotification: false
+          })
+          .then(function(imageStream) {
+            $scope.imageStream = imageStream;
+            DataService
+              .get("imagestreamtags", imageStream.metadata.name + ":" + createDetails.imageTag, {namespace: createDetails.namespace}, {
+                  errorNotification: false
+              })
+              .then(function(imageStreamTag){
+                $scope.imageStreamTag = imageStreamTag;
+                $scope.validationPassed = true;
+                $scope.resource = imageStreamTag;
+                createDetails.displayName = $filter('displayName')(imageStreamTag);
+              }, function(e){
+                alertInvalidImageTag(createDetails.imageTag, e);
+              });
+          }, function(e) {
+            if(e.status === 403) {
+              alertInaccessibleNamespace(createDetails.namespace, e);
```

Comment:
You might have access to the namespace, but not the image stream. I'd change the message to say specifically that you don't have access to the image stream or template.